drivers/at86rf2xx/at86rf2xx_netdev.c

```diff
@@ -672,6 +672,25 @@ static void _isr_send_complete(at86rf2xx_t *dev, uint8_t trac_status)
             }
```

Comment:
Can't comment on the specific line, but just above, `trac_status` only applies to extended mode but  `TRAC_STATUS = 7 (INVALID)` when `RX_AACK and TX_ARET` are started. If they where previouslt set the callback will never be executed. IMO for basic mode it should call the callback on `AT86RF2XX_TRX_STATE__TRAC_INVALID` or `AT86RF2XX_TRX_STATE__TRAC_SUCCESS`.

Reply:
Taken from `at86rf233` datasheet p70

Reply:
I completely forgot that! Sure, that function should never be called in Basic Mode. We can just directly issue a "TX_COMPLETE"

Reply:
done!